TemplateProject/.vimrc

```diff
@@ -0,0 +1 @@
+let g:ale_javascript_prettier_options = '--config .prettierrc'
```

Comment:
Does this not get set by default? We might let users set this themselves locally. I don't feel strongly about this though.

Reply:
Unfortunately it doesn't. Given that there's a project-specific style, I don't think there's any harm in including this for ALE users. I think it's currently up to the tools to know about and support prettier config files.